ctru-rs/src/lib.rs

```diff
@@ -28,6 +29,7 @@ pub fn init() {
 
     use std::panic::PanicInfo;
 
+    #[cfg(not(test))]
```

Comment:
nit: this panic hook setup could be in a separate function so we only need one `cfg(not(test))` and it doesn't get interwoven in `init`.